### DISCUSSION TITLE
How do I find the user_id?

### DISCUSSION THREAD

**Discussion by ValiVKhan** [mmohades/Venmo#32] [Q&A] [answered]
I am trying to make a test payment through this API however am not able to find a way to get the user_id. How do I find the value for the user_id?

Thank you

**Comment by mmohades** [comment, marked as answer]
You can saerch for users or get users by their username, then access the user_id

Here is an example:
```python
users = client.user.search_for_users(query="Peter")
for user in users:
   print(user.id)
```
or you can get a user by their username:
```python
user = client.user.get_user_by_username(username="andrew")
print(user.id)
```
Note that the above method may return `None` if it couldn't find the user with the given username.

**Reply by ValiVKhan** [reply]
Thank you for the help!!!

**Reply by mmohades** [reply]
Sure! Did you get it working?

**Reply by ValiVKhan** [reply]
Yes! I will be asking more questions as I run into more problems in the future. Thank you for being such a fast responder!

**Reply by ValiVKhan** [reply]
Ok, so I am receiving an error when I try to do a send_money

This is the error:
There is not enough balance on the default payment method to complete the transaction.
hint: Use other payment methods like
send_money(amount, tr_note, target_user_id, funding_source_id=other_payment_id_here)
or transfer money to your default payment method.

Could please give me an example of how to fill out "(amount, tr_note, target_user_id, funding_source_id=other_payment_id_here)" or tell me how to add balance to my default payment method?

Thank you
